README.md

```diff
@@ -6,7 +6,7 @@
 
 <br />
 
-> This branch is **unstable** and is in **active development**.  
+> This branch is **unstable** and is in **active development**.
```

Comment:
Did you remove double space in the end?

Reply:
Oh sorry, it probably is my editor configured to remove trailing whitespace. I can add it back if you want.

Reply:
Trailing whitespace in markdown is line break, it's make difference. 😢

Reply:
Ooops, did not know that. I'll fix it and fix my editor settings. Sorry.